ford_fusion_sehybrid_2019/drivers.launch

```diff
@@ -71,6 +73,14 @@ If not using simulated drivers they are activated if the respective mock argumen
     <arg name="device_ip" value="192.168.1.201"/>
   </include>
 
+    <!-- Lightbar Driver Node -->
+    <include if="$(arg lightbar_driver)" file="$(find lightbar_driver)/launch/lightbar_driver_node.launch">
+      <arg name="host_name" value="192.168.88.28" />
+      <arg name="port" value="80" />
+      <arg name="user" value="root" />
+      <arg name="password" value="00000000" />
```

Comment:
I know its pretty low risk to expose the light bar password, but we should be in the habit of securing all passwords on the system. Please move the user and password to the VehicleCalibration repo which is private. The SSC driver has an example of this above.